src/Core/LanguageService.fs

```diff
@@ -303,6 +303,45 @@ module LanguageService =
                 ()
             )
 
+
+    module FakeSupport =
+        open DTO.FakeSupport
+        
+        let logger = ConsoleAndOutputChannelLogger(Some "FakeTargets", Level.DEBUG, None, Some Level.DEBUG)
+        
+        let targetsInfo (fn:string) =
+            match client with
+            | None ->
+                Promise.empty
+            | Some cl ->
+                Environment.dotnet
+                |> Promise.bind (fun dotnetRuntime ->
+                    match dotnetRuntime with
+                    | Some r ->
+                        let extensionPath = VSCodeExtension.ionidePluginPath()
+                        let runtimePath = extensionPath + "/.fakeruntime"
```

Comment:
I don't think this is a good way to do it. Here is what this is about:

In order to list target's we need to run fake. So this is the folder we download the `fake.dll` portable latest version into.
The following problems arise:
- What if there is a newer version? How can a user ugprade?
- In development, this path is `/release/.fakeruntime` so it potentially gets bundled into the extension? Is this what we want?

The following questions are open:
- Is there a better path to put lazily loaded files into?
- Should we bundle `fake.dll` like we do for `fsautocomplete.dll`? -> I think we don't want to.

Reply:
Maybe this should be per-repository, rather than global? Maybe using `.fake` folder?

Reply:
Yes I guess you try to address 

> What if there is a newer version? How can a user upgrade? 

In per-repository setting user can `git clean` and then stuff should work again. On the other side when people encounter issues they might take a while to try `git clean`, so I'm not sure this even addresses that (in the sense that it will still cost a lot of people a lot of debugging time)

However, I'm working towards a model where FAKE scripting works well with Ionide. In this "worldview" there is no repository. Maybe I'm just editing single scripts. So now ionide downloads the fake package (~20MB) for each script... 
Additionally, performance might be an issue: When opening a new script Ionide starts downloading. Depending on your current internet speed it will take until everything works. While this is true for packages as well, those are at least in the nuget package cache.

If we decide we can live with those issues I'm strictly against the `.fake` folder. That folder is completely controlled by `fake`. All contents are basically implementation detail and subject to change. So we would use an `.ionide` folder. This also makes people report issues to the correct place in case they realize where their harddrive space is going ;)